README.md

```diff
@@ -25,3 +25,83 @@ $$ T(n) =
         13 T\left(\frac{n}{13}\right) + 2n & n > 1
     \end{cases}
 $$
+
+---
+
+Answers:
+
+1.
+
+$$
+T(n) = T(\frac{n}{13}) + 5 \\
+= 13(T(\frac{n}{169}) + \frac{5}{13}) + 5 \\
+= 13T(\frac{n}{169}) + 10 \\
+... \\
+= 13^{(i - 1)}T(n \cdot 13^{-i}) + 5i
+$$
+
+Let $i = \log_{13}(n)$:
+
+$$
+13^{\log_{13}(n) - 1}T(n \cdot 13^{-\log_{13}(n)}) + 5\log_{13}(n) \\
+= \frac{1}{13}n \cdot T(n \cdot \frac{1}{n}) + 5\log_{13}(n) \\
+= 5 \log_{13}(n) + \frac{1}{13}T(1) \\
+= 5 \log_{13}(n) + \frac{1}{13}n
+$$
+
+$5\log_{13}(n) + \frac{1}{13}n \in \Theta(n)$
```

Comment:
This one is not correct -- is there really as much work happening as for the next recurrence relation?